apps/ssr-e2e/src/integration/app.spec.ts

```diff
@@ -20,4 +20,18 @@ describe('@rx-angular/template universal rendering', () => {
       )
     });
   });
+
+  describe('RxFor', () => {
+    it('should display green and purple text', () => {
+      cy.request('http://localhost:4200').its('body').then(
+        html => {
+          console.log(html)
```

Comment:
Not sure what's your process is for this repo but you might want to remove this.